### PR TITLE
#65: Standardize localization

### PR DESCRIPTION
**Pull request by hani-ibrahim**
working on #65

### CODE REVIEW

File: AtlasUI/AtlasUI/AddressFormViewController.swift [hunk 1]
```diff
@@ -64,12 +64,12 @@ class AddressFormViewController: UIViewController, CheckoutProviderType {
 extension AddressFormViewController {
 
     private func configureNavigation() {
-        let saveButton = UIBarButtonItem(title: Localizer.string("Save"),
+        let saveButton = UIBarButtonItem(title: Localizer.string("button.title.save"),
```
Comment (bonkey) [inline AtlasUI/AtlasUI/AddressFormViewController.swift]: Why doesn't it start with a _C_apital letter?

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**